src/app/treelist/treelist.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
+import { TreeNode, TREE_ACTIONS, KEYS, IActionMapping, TreeComponent } from 'angular2-tree-component';
```

Comment:
Actually, KEYS can be removed now. I was using this for when the user hits enter, but removed the functionality.

keys: {
    [KEYS.ENTER]: (tree, node, $event) => alert(`This is ${node.data.name}`)
  }